components/k8s-model-server/images/releaser/components/workflows.libsonnet

```diff
@@ -247,6 +247,11 @@
           template: "test-tf-serving",
           dependencies: ["deploy-tf-serving"],
         },
+        {
+          name: "deploy-spark-job",
```

Comment:
Why did you modify this file?
I think this E2E test is specific for K8s model server.